recipes/pyturbo/meta.yaml

```
@@ -0,0 +1,43 @@
+{% set name = "pyturbo" %}
+{% set version = "0.0.1" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  - url: https://github.com/twiinit/pyturbo/archive/refs/tags/v{{ version }}.tar.gz
+    sha256: 4e777fd460107ad0d1f948e8f230976381c689cc1e6ac2c754670c51698d8615
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed --no-cache-dir -vvv"
```

```suggestion
  script: {{ PYTHON }} -m pip install . -vv
```
This is sufficient, the rest will be done by conda-forge.

---

Even the `--no-deps` option has to be removed since I use `pip` to install in a `conda` env?

---

Yes, that is now governed by conda-forge in the build environment. So you don‘t have to set it, it is already set for you. 
If you’re very hesitant you can still set it but there shouldn’t be any difference.

---

Just to add: This is, for example, the output of Grayskull which we use for recipes nowadays pretty much all the time.

```yaml
{% set name = "pyturbo" %}
{% set version = "18.7.27" %}

package:
  name: {{ name|lower }}
  version: {{ version }}

source:
  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/pyturbo-{{ version }}.tar.gz
  sha256: 11aea3102204a16e4f0a05022f1f231ca1c8d40d069b55a64b2eec96f99dc277

build:
  noarch: python
  script: {{ PYTHON }} -m pip install . -vv
  number: 0

requirements:
  host:
    - pip
    - python
  run:
    - numpy
    - opencv
    - python
    - six
    - tensorflow

test:
  imports:
    - pyturbo
  commands:
    - pip check
  requires:
    - pip

about:
  home: https://github.com/aikudexiaohai/pyturbo
  summary: This project contains a variety of different python packages
  license: BSD-2-Clause
  license_file: PLEASE_ADD_LICENSE_FILE

extra:
  recipe-maintainers:
    - ADD_YOUR_GITHUB_ID_HERE
```